orchagent/port/porthlpr.cpp

```diff
@@ -233,6 +240,11 @@ std::string PortHelper::getAdminStatusStr(const PortConfig &port) const
     return this->getFieldValueStr(port, PORT_ADMIN_STATUS);
 }
 
+std::string PortHelper::getDampingAlgoStr(const PortConfig &port) const
```

Comment:
not sure if Str is reqired in function naming, you already knw what type is returned

Reply:
Sure, I removed Str from the function name.